libraries/AP_BattMonitor/AP_BattMonitor_INA3221.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+/*
+ *
+
+ * DataSheet: https://www.ti.com/lit/ds/symlink/ina3221.pdf?ts=1597369254046
+ */
+
+// The INA3221takestwo measurementsfor eachchannel:one for shuntvoltageand one for bus voltage.Eachmeasurementcan be independentlyor sequentiallymeasured,basedon the modesetting(bits2-0 in theConfigurationregister).Whenthe INA3221is in normaloperatingmode(thatis, the MODEbits of theConfigurationregisterare set to 111),the devicecontinuouslyconvertsa shunt-voltagereadingfollowedby abus-voltagereading.Thisprocedureconvertsone channel,and thencontinuesto the shuntvoltagereadingofthe nextenabledchannel,followedby the bus-voltagereadingfor that channel,and so on, untilall enabledchannelshavebeenmeasured.The programmedConfigurationregistermodesettingappliesto all channels.Any channelsthat are not enabledare bypassedin the measurementsequence,regardlessof modesetting.
```

Comment:
Line breaks, please? I'm not one to normally complain, but not only is this missing line breaks, it's missing a ton of spaces, and is just very hard to read.